document-store/src/main/java/org/hypertrace/core/documentstore/postgres/PostgresCollection.java

```diff
@@ -130,7 +130,26 @@ public CreateResult create(Key key, Document document) throws IOException {
 
   @Override
   public BulkUpdateResult bulkUpdate(List<BulkUpdateRequest> bulkUpdateRequests) throws Exception {
-    throw new UnsupportedOperationException();
+
+    var requestsWithFilter =
```

Comment:
nit: replace `var` with actual types for readability.

Reply:
Done